PWGCF/FemtoUniverse/Tasks/femtoUniversePairTaskTrackTrackMultKtExtended.cxx

```diff
@@ -241,28 +242,64 @@ struct femtoUniversePairTaskTrackTrackMultKtExtended {
 
   HistogramRegistry sphericityRegistry{"SphericityHisto", {}, OutputObjHandlingPolicy::AnalysisObject, true, true};
 
-  /// TPC Pion/Kaon/Proton Sigma selection
+  /// TPC Pion/Kaon/Proton Sigma selection (general)
   bool IsNSigma(float mom, float nsigmaTPC, float nsigmaTOF)
   {
     // |nsigma_TPC| < 3 for p < 0.5 GeV/c
     // |nsigma_combined| < 3 for p > 0.5
 
     // using configurables:
-    // ConfTOFPtMin -- momentum value when we start using TOF; set to 1000 if TOF not needed
-    // ConfNsigmaTPC -> TPC Sigma for momentum < ConfTOFPtMin
-    // ConfNsigmaCombined -> TPC and TOF Sigma (combined) for momentum > ConfTOFPtMin
+    // ConfTOFpMin -- momentum value when we start using TOF; set to 1000 if TOF not needed
+    // ConfNsigmaTPC -> TPC Sigma for momentum < ConfTOFpMin
+    // ConfNsigmaCombined -> TPC and TOF Sigma (combined) for momentum > ConfTOFpMin
 
-    if (mom < twotracksconfigs.ConfTOFPtMin) {
+    if (mom < twotracksconfigs.ConfTOFpMin) {
       return TMath::Abs(nsigmaTPC) < twotracksconfigs.ConfNsigmaTPC;
     } else {
       return TMath::Hypot(nsigmaTOF, nsigmaTPC) < twotracksconfigs.ConfNsigmaCombined;
     }
   }
 
+  /// TPC Kaon Sigma selection (stricter cuts for K+ and K-)
+  bool IsKaonNSigma(float mom, float nsigmaTPCK, float nsigmaTOFK)
+  {
+    if (twotracksconfigs.IsKaonNsigma == true) {
+      if (mom < 0.5) {
+        if (TMath::Abs(nsigmaTPCK) < 2) {
```

Comment:
you can again shorten this function with forms like `return TMath::Abs(nsigmaTPCK) < 2);`.
Similarly as you did in IsNSigma().